tests/auth.test.ts

```diff
@@ -218,20 +218,6 @@ describe("Authentication Tests", () => {
       expect(res2).toHaveProperty("factory", FACTORY_ADDRESS);
     });
 
-    test("getWallets works with options.authKey", async () => {
```

Comment:
This test has the exactly same name with the previous one, so I removed it.

I think it’s trying to test a default value salt:0, but there’s already a salt:123 created from the previous tests. We don’t have a way to reset the smart wallet response array yet.